### PR TITLE
Wallet img switch

### PR DESCRIPTION
**Pull request by MandatoryMania**
The previous wallet button image had a black outline instead of white; hence when the button was shaded the entire object appeared dark.  This change changes this outline and also the design of the wallet.  The change passed the initial developer testing.

*Test Plan, for internal #discovery testing only*

-Note button default appearance: should be white on black
-Note appearance while hovering over button: should be white on gray
-Note appearance after clicking, and triggered action: should be black on white with panel opened
-Note appearance upon clicking again, and triggered action: should be default with panel closed
-Compare above with other buttons:  should be analogous
-Repeat all of above with: Settings -> Advanced Menus, Settings -> Developer Menus, Developer ->
 Debug defaultScripts.js in order to validate button loading:  should be identical to above

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -319,7 +319,7 @@ static const int THROTTLED_SIM_FRAME_PERIOD_MS = MSECS_PER_SECOND / THROTTLED_SI
 
 static const uint32_t INVALID_FRAME = UINT32_MAX;
 
-static const float PHYSICS_READY_RANGE = 3.0f; // how far from avatar to check for entities that aren't ready for simulation
+static const float PHYSICS_READY_RANGE = 1.0f; // how far from avatar to check for entities that aren't ready for simulation
```
Comment (howard-stearns) [inline interface/src/Application.cpp]: Looks like something left over from merging issues? In any case I don't think this belongs in this PR? Please revert it.

File: libraries/physics/src/EntityMotionState.cpp [hunk 2]
```diff
@@ -531,7 +531,7 @@ void EntityMotionState::sendUpdate(OctreeEditPacketSender* packetSender, uint32_
         }
 
         if (!_body->isStaticOrKinematicObject()) {
-            const float DYNAMIC_LINEAR_VELOCITY_THRESHOLD = 0.05f;  // 5 cm/sec
+            const float DYNAMIC_LINEAR_VELOCITY_THRESHOLD = 0.02f;  // 5 cm/sec
```
Comment (howard-stearns) [inline libraries/physics/src/EntityMotionState.cpp]: same

File: libraries/physics/src/PhysicsEngine.cpp [hunk 3]
```diff
@@ -441,7 +441,7 @@ const CollisionEvents& PhysicsEngine::getCollisionEvents() {
     while (contactItr != _contactMap.end()) {
         ContactInfo& contact = contactItr->second;
         ContactEventType type = contact.computeType(_numContactFrames);
-        const btScalar SIGNIFICANT_DEPTH = -0.002f; // penetrations have negative distance
+        const btScalar SIGNIFICANT_DEPTH = -0.008f; // penetrations have negative distance
```
Comment (howard-stearns) [inline libraries/physics/src/PhysicsEngine.cpp]: same

**Comment by MandatoryMania** [comment]
Those were 3 small changes to constants I did for the build test before I was hired.  I should have created a new fork!
I can simply do another push with those constants out back.
Reverting is something I don't do too often...may need some help.




Sent from my T-Mobile 4G LTE Device

-------- Original message --------

**Comment by MandatoryMania** [comment]
I meant "put back"


Sent from my T-Mobile 4G LTE Device

-------- Original message --------
From: hqmail <hqmail@comcast.net> 
Date: 8/31/17  3:31 PM  (GMT-08:00) 
To: highfidelity/hifi <reply@reply.github.com>, highfidelity/hifi <hifi@noreply.github.com> 
Cc: Author <author@noreply.github.com> 
Subject: Re: [highfidelity/hifi] Wallet img switch (#11292) 


    
Those were 3 small changes to constants I did for the build test before I was hired.  I should have created a new fork!
I can simply do another push with those constants out back.
Reverting is something I don't do too often...may need some help.




Sent from my T-Mobile 4G LTE Device

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2990/HighFidelity-Beta-PR11292-6f1bb6abe28df76956db1be8e55e007f745fa3e7.dmg

**Comment by zfox23** [comment]
[art: wallet icon has black outline instead of white](https://trello.com/c/H3mdx1wK/311-art-wallet-icon-has-black-outline-instead-of-white)

**Comment by MandatoryMania** [comment]
Pushing same wallet button solution with previous changed constants reverted to default values. 
Will issue new PR.  Easiest method to do this; after much appreciated convo with David.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2999/HighFidelity-Beta-PR11292-6f1bb6abe28df76956db1be8e55e007f745fa3e7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2999/HighFidelity-Beta-PR11292-6f1bb6abe28df76956db1be8e55e007f745fa3e7.exe

**Pull request closed without merging**